src/main/java/com/sap/sgs/phosphor/fosstars/tool/github/AbstractRatingCalculator.java

```diff
@@ -43,9 +49,13 @@
    * Initializes a new calculator.
    *
    * @param github An interface to GitHub.
+   * @param nvd An interface to NVD.
    */
-  AbstractRatingCalculator(GitHub github) {
-    this.github = Objects.requireNonNull(github, "Oh no! An interface to GitHub can't be null!");
+  AbstractRatingCalculator(GitHub github, NVD nvd) {
+    Objects.requireNonNull(github, "Oh no! An interface to GitHub can't be null!");
```

Comment:
can be written like this right?
```
this.github = Objects.requireNonNull(github, "Oh no! An interface to GitHub can't be null!");
this.nvd = Objects.requireNonNull(nvd, "Oh no! An interface to NVD can't be null!");
```

Reply:
Right. Your code may leave a partly-initialized instance. That's not a bid deal though. But I would like to check all parameters before using them. Just trying to build a good habit.